testing/test_assertrewrite.py

```diff
@@ -1369,12 +1371,15 @@ def test_fails():
                 assert False, "assert with message"
             """
         )
-        result = testdir.runpytest()
-        result.stdout.fnmatch_lines(
-            "*Assertion Passed: a + b == c + d (1 + 2) == (3 + 0) at line 7*"
+        rec = testdir.inline_run()
+        calls = rec.getcalls("pytest_assertion_pass")
```

Comment:
@asottile this might be useful too, using `HookRecorder` objects to get the hook call directly